static/app/components/actions/ignore.tsx

```diff
@@ -236,9 +236,8 @@ const IgnoreActions = ({
       </IgnoreButton>
       <DropdownMenuControl
         size="sm"
-        trigger={({props: triggerProps, ref: triggerRef}) => (
+        trigger={triggerProps => (
           <DropdownTrigger
-            ref={triggerRef}
```

Comment:
The `triggerProps` from `useOverlay` contains a `ref` prop, so we don't need the second argument.